Assembly/Scripts/Characters/Shifters/BaseShifter.cs

```diff
@@ -74,6 +76,7 @@ protected IEnumerator WaitAndBecomeHuman(float time)
             StartCoroutine(WaitAndDie());
             yield return new WaitForSeconds(2f);
             _inGameManager.SpawnPlayerAt(false, BaseTitanCache.Neck.position);
+            StartCoroutine(((Human)_inGameManager.CurrentCharacter).WaitAndTransformFromShifter(PreviousHumanGas, PreviousHumanWeapon));
```

Comment:
use CurrentCharacter.StartCoroutine instead of doing StartCoroutine. The coroutine should belong to the human not the game manager